key/key_test.go

```diff
@@ -73,7 +73,7 @@ func Test_RoleName(t *testing.T) {
 				"blue",
 				"green",
 			},
-			ExpectedResult: "role-org-ae04e382ff1b455a454bfde83bdda9dc8d077649",
+			ExpectedResult: "role-org-90b351b2fd11e3f6adabda139ebb28a73a8a6997c1db1a9f2214dd2775e9e953378a10003d7a42bc1efd0ee970f1a380ee9a4a39e1ef6b1ec79700d659fc77ba",
```

Comment:
Since the crypto package changed the role keys change. I tested this on `gauss`. Created a cluster and then deployed a modified version of `cert-operator`. The old roles stay for now. This little garbage doesn't harm. We can clean it up if it annoys someone deeply. The new role names are simply used and new roles created. For them are new certificates created. Since everything is part of the same PKI and root CA and the roles keep their associated organisations and common names this is like rotating certificates, which happens every two weeks or so anyway. Created new Node Pools once the role names changed and the whole cluster was just fine like nothing happened. All good IMO.